extension/scripts/features/total-portfolio-value/ttTotalPortfolioValue.css

```diff
@@ -0,0 +1,15 @@
+.tt-total-stock-value {
+	display: inline-block;
+    margin: 0 0 0 9px;
+	color: var(--default-content-title-color);
```

Comment:
This should not be needed since it's inside of an element that already has this color.